lib/gui/models/store.js

```diff
@@ -49,6 +49,7 @@ const DEFAULT_STATE = Immutable.fromJS({
     errorReporting: true,
     unmountOnSuccess: true,
     validateWriteOnSuccess: true,
+    disableUpdates: !packageJSON.updates.enabled,
```

Comment:
Is it maybe a bit 'contradictory' that "internally" (i.e. in code) we refer to it as `disableUpdates` but "externally" (i.e. in `package.json`) we refer to it as `updates.enabled`?
Although I guess as long as we're consistent it should be okay.